### PR TITLE
include qos_profile_* constants in documentation

### PR DESCRIPTION
**Pull request by goekce**
This PR makes `qos_profile_*` constants in [QoS documentation](http://docs.ros2.org/foxy/api/rclpy/api/qos.html#rclpy.qos.QoSProfile) visible. Their usage is recommended on https://index.ros.org/doc/ros2/Releases/Release-Dashing-Diademata/#rclpy but hiding them in the documentation may confuse users.

The problem is that sphinx-autodoc currently does not include module-level variables, unless these variables include `#:` style comments. See https://github.com/sphinx-doc/sphinx/issues/1063

I did not look into source files other than `qos.py`, because including variables using this workaround is tedious. It is better to wait for https://github.com/sphinx-doc/sphinx/issues/1063

Note that this PR is only for foxy. 

fixes the issue: https://github.com/ros2/docs.ros2.org/issues/49

### CODE REVIEW

File: rclpy/docs/source/conf.py [hunk 1]
```diff
@@ -194,9 +194,9 @@
 
 # -- Extension configuration -------------------------------------------------
 
-autoclass_content = 'both'
+#autoclass_content = 'both'
```
Comment (jacobperron) [inline rclpy/docs/source/conf.py:197]: Why was this setting commented out? Seems like we could leave it how it was.

File: rclpy/rclpy/qos.py [hunk 2]
```diff
@@ -381,20 +381,23 @@ class LivelinessPolicy(QoSPolicyEnum):
 # Alias with the old name, for retrocompatibility
 QoSLivelinessPolicy = LivelinessPolicy
 
+# module-level data members are not automatically documented by Sphinx. Use an
+# empty ``#:`` as a workaround. See
+# https://github.com/sphinx-doc/sphinx/issues/1063
```
Comment (jacobperron) [inline rclpy/rclpy/qos.py:386]: Rather than appending an empty `#:` as a workaround, I think it would be better to add some content to the doc comment.

It could be something minimal, for example,

```python
#: A good QoS profile for sensor data, using best effort reliability.
qos_profile_sensor_data = QoSProfile(**_rclpy.rclpy_get_rmw_qos_profile(
    'qos_profile_sensor_data'))
```

Note, it looks like Sphinx is happy to have the comment appearing the line before the variable.

File: rclpy/docs/source/conf.py [hunk 3]
```diff
@@ -194,9 +194,9 @@
...
 
 autodoc_default_options = {
-    'members': None,
-    'undoc-members': True,
+    'members': True,  # document members
```
Comment (jacobperron) [inline rclpy/docs/source/conf.py:200]: I've tried reverting this change and the `qos_profile_*` variables still appear in the docs. I think it is the addition of the `#:` comments that is making them appear, and this line has no affect on them. I think we can undo this change.

**Review by jacobperron — CHANGES_REQUESTED** [review]
Thanks for looking into it! I've left feedback below.

**Comment by jacobperron** [comment]
@goekce Note, we enforce that contributors "sign" the Developer Certificate of Origin (DCO)](https://developercertificate.org/) (hence the failing DCO check). Please take a look at our [developer guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) and "sign-off" on all of your commits when contributing to the ROS 2 project.

File: rclpy/docs/source/conf.py (hunk 1 above)
Comment (goekce) [reply rclpy/docs/source/conf.py:197]: sorry, it seems that I forgot to revert it back while experimenting!

File: rclpy/docs/source/conf.py (hunk 3 above)
Comment (goekce) [reply rclpy/docs/source/conf.py:200]: >  I think it is the addition of the `#:` comments that is making them appear, and this line has no affect on them.

True, still: `None` and `True` keywords have the same meaning. I would stick to one of them to avoid confusion. What do you think?

File: rclpy/rclpy/qos.py (hunk 2 above)
Comment (goekce) [reply rclpy/rclpy/qos.py:386]: This is a good idea! I never used these profiles, but can try to get some info from rmw docs.

File: rclpy/docs/source/conf.py (hunk 3 above)
Comment (goekce) [reply rclpy/docs/source/conf.py:200]: I think it makes more sense to include this minor change in another commit.

Comment (jacobperron) [reply rclpy/docs/source/conf.py:200]: > I think it makes more sense to include this minor change in another commit.

Sounds good to me.

**Review by jacobperron — APPROVED** [review]
Thanks for the improvements!
